docs/data-sources/connector.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_connector Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Connectors provide connectivity to Remote Networks. This resource type will create the Connector in the Twingate Admin Console, but in order to successfully deploy it, you must also generate Connector tokens that authenticate the Connector with Twingate. For more information, see Twingate's documentation https://docs.twingate.com/docs/understanding-access-nodes.
+---
+
+# twingate_connector (Data Source)
+
+Connectors provide connectivity to Remote Networks. This resource type will create the Connector in the Twingate Admin Console, but in order to successfully deploy it, you must also generate Connector tokens that authenticate the Connector with Twingate. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/understanding-access-nodes).
```

Comment:
This description doesn't apply since this is from the resource description. Let's update the description to the following short version:

`Connectors provide connectivity to Remote Networks. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/understanding-access-nodes).`

Reply:
updated